### PR TITLE
fix(build): cross platform cp

### PR DESCRIPTION
**Pull request by gilhanan**
Fix https://github.com/ngx-translate/core/issues/416

* Install npm package 'cash-cp' as devDependency

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,6 +46,7 @@
     "@types/jasmine": "2.5.41",
     "@types/node": "7.0.4",
     "awesome-typescript-loader": "3.0.0-beta.18",
+    "cash-cp": "^0.2.0",
```
Comment (deeg) [inline package.json]: This should probably be hardcoded to a specific version like the other dev dependencies.

Comment (gilhanan) [inline package.json]: Thanks, pinned to specific version

**Comment by SamVerschueren** [comment]
Just wondering what the cause of this issue is. Which plugin/dependency uses `cp` internally?

**Comment by ocombe** [comment]
This PR was for v5, but I changed the package.json and it no longer uses cp in the npm commands, so I'm closing the PR since it's no longer an issue :)

**Pull request closed without merging**